SwiftStylable/Classes/Style/StyleSets/StyleSetBase.swift

```diff
@@ -85,16 +86,29 @@ public class StyleSetBase : StyleSet {
         return color
     }
     
-    internal func parseFont(data:[String:Any]?, defaultName:String? = nil, defaultSize:CGFloat? = nil)->UIFont? {
-        let name = data?["name"] as? String ?? defaultName
-        let sizeAny = data?["size"]
+    internal func dimensionFromValue(_ value:Any?)->CGFloat? {
         var size:CGFloat?
-        if sizeAny != nil {
-            size = sizeAny as? CGFloat
-            if size == nil {
-                print("WARNING: Style definition for '\(self.name)' has a font size of type String, change to a Number in the styles.plist")
+        guard let value = value else {
+            return nil
+        }
+        
+        if let value = value as? String{
+            if let dimentionNamed = self._dimensionCollection.dimensionHolderNamed(value){
+                size = dimentionNamed.size
+            } else {
+                print("WARNING: Dimention named \(value) could not be found/parsed")
```

Comment:
Dimension ipv Dimention. En: misschien kunnen we nog wat aan de melding toevoegen. Voorheen was er een melding als de fontSize een string was, want dan was de gebruiker ws gewoon vergeten de variabele op 'number' te zetten in de plist. Nu is een string natuurlijk opeens toegestaan in een dimension, maar het kan nog steeds dat er een verkeerd veld type wordt gebruikt. Misschien ook handig om de style(set) name mee te loggen, anders zoek je je een breuk.